lighthouse-cli/test/smokehouse/byte-efficiency/expectations.js

```diff
@@ -10,6 +10,68 @@
  */
 module.exports = [
   {
+    artifacts: {
+      ScriptElements: [
+        {
+          type: null,
+          src: null,
+          async: false,
+          defer: false,
+          source: 'head',
+          devtoolsNodePath: '2,HTML,0,HEAD,3,SCRIPT',
+        },
+        {
+          type: 'application/javascript',
+          src: 'http://localhost:10200/byte-efficiency/script.js',
+          async: false,
+          defer: false,
+          source: 'head',
+          devtoolsNodePath: '2,HTML,0,HEAD,5,SCRIPT',
+        },
+        {
+          type: null,
+          src: null,
+          async: false,
+          defer: false,
+          source: 'body',
+          devtoolsNodePath: '2,HTML,1,BODY,0,DIV,3,SCRIPT',
+        },
+        {
+          type: null,
+          src: null,
+          async: false,
+          defer: false,
+          source: 'body',
+          devtoolsNodePath: '2,HTML,1,BODY,3,SCRIPT',
+        },
+        {
+          type: null,
+          src: 'http://localhost:10200/byte-efficiency/delay-complete.js?delay=8000',
+          async: true,
+          defer: false,
+          source: 'body',
+          devtoolsNodePath: '2,HTML,1,BODY,1438,SCRIPT',
```

Comment:
This alternated between 1438 / 1437, until I changed the script to be a 8s delay (there's a 7s timer that adds an element)